vision_datasets/common/data_manifest.py

```diff
@@ -447,6 +451,17 @@ def create_dataset_manifest(coco_file_path_or_url, data_type, container_sas_or_r
 
         file_reader.close()
 
+        if data_type == DatasetTypes.IMCAP:
```

Comment:
Can you add a check in IrisManifestAdaptor to reject this type?

Reply:
Done!